src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random r = new Random();

```

Comment:
```suggestion
        Random random = new Random();

```

dont forget about informative variable names